docs/_docs/best-practices/general/general.md

```diff
@@ -28,6 +29,157 @@ Discover essential FinOps best practices to optimize cost efficiency and governa
 
 ---
 
+## Resource inventory summary
+
+### Query: Count of all resources
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+Resources
+| summarize count()
+```
+
+<br>
+
+### Query: Count of all resources per subscription
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope per subscription.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+resources
+| summarize Count=count(id) by subscriptionId
+| order by Count desc
```

Comment:
nit: I'm on the fence as to whether we should put sorting in the queries. We don't want unnecessary load on the server. I get it may be useful in some scenarios, but I'd probably leave that to the user to decide on their own unless sorting is critical for the scenario.